patches/server/0842-Patch-Add-new-overload-to-PersistentDataContainer-ha.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: u9g <winworkswow@gmail.com>
+Date: Mon, 3 Jan 2022 23:32:42 -0500
+Subject: [PATCH] [Patch] Add new overload to PersistentDataContainer#has
+
+Adds the new overload: PersistentDataContainer#has(NamespacedKey key)
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java b/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
+index 2c59f09a9261a1690951161fd856a5848d9885b7..987c8144738c6e016f5b60b41806bb2a9afbd1a2 100644
+--- a/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
++++ b/src/main/java/org/bukkit/craftbukkit/persistence/CraftPersistentDataContainer.java
+@@ -161,5 +161,15 @@ public class CraftPersistentDataContainer implements PersistentDataContainer {
+     public void clear() {
+         this.customDataTags.clear();
+     }
++
++    public boolean has(@NotNull NamespacedKey key) {
++        Validate.notNull(key, "The provided key for the custom value was null");
++
++        Tag value = this.customDataTags.get(key.toString());
```

Comment:
This entire section can be simplified to
```java
return this.customDataTags.get(key.toString()) != null;
```

Reply:
Done ;D